### ISSUE TITLE
Correct path to mount volumes into the builder image?

### ISSUE THREAD

**Issue by vinayluzrao** [GoogleCloudPlatform/buildpacks#126]
My source code directory has a symlink that points to a path not contained in its subtree. Therefore, when I try building, the symlink that is copied into the builder is dead (dangling). To solve this, I thought of using the `--volume` flag, of the `pack build` command, to copy over the directory that my symlink references:

```
--volume stringArray       Mount host volume into the build container, in the form '<host path>:<target path>[:<mode>]'.
                                 Repeat for each volume in order,
                                   or supply once by comma-separated list
```

But, I don't know the correct `target path` for the command, and I believe this isn't documented anywhere. Can someone tell me what it is?

**Comment by vinayluzrao** [comment]
> The help text was updated in [buildpacks/pack#986](https://github.com/buildpacks/pack/pull/986). Does the new version provide more clarity?
> 
> I believe the target path would just be an absolute path you want to mount the directory at, taken verbatim, e.g. `/my/path` would mount it at `/my/path`. For the symlinks, repeating the same directory path as you provide for in might work.
> 
> It looks like this issue isn't related to this repo specifically. The `pack` tool is maintained under [buildpacks/pack](https://github.com/buildpacks/pack), please open the issue there if it continues to be a problem.

I had assumed it relates to this repo because I thought that each builder implementation has their own working directory structure. And so, if you want to mount a volume, you have to know where the source code is located so that the volume is referenced properly. Is that not the case?

**Comment by lukasberger** [comment]
No, that's not the case. The platform (pack in this case) specifies the the working directory through CNB_APP_DIR (see https://github.com/buildpacks/spec/blob/main/platform.md).

The working directory is also typically mounted as a volume, so you will have to choose a different directory to mount the second volume as.
